### PR TITLE
[CCXDEV-10823] Adding Glitchtip confiuration to notification service

### PR DESCRIPTION
**Pull request by Jakub007d**
# Description
Adding configuration for notification service to be able to send error mesages to GlitchTip.

Fixes # [CCXDEV-10823](https://issues.redhat.com/browse/CCXDEV-10823)

## Type of change

- New feature (non-breaking change which adds functionality)
- Configuration update

## Testing steps

Tested locally with config.toml file and also by exporting the env var

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: differ/differ.go [hunk 1]
```diff
@@ -996,6 +996,7 @@ func Run(config conf.ConfigStruct, cliFlags types.CliFlags) int {
 	if cliFlags.InstantReports {
 		notificationType = types.InstantNotif
 	}
+	log.Error().Msg("TEST ERROR FOR NOTIFICATION SERVICE IN DIFFER")
```
Comment (epapbak) [inline differ/differ.go:999]: remove, there will be errors already, so it's not needed

File: cmd/ccx-notification-service/main.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func main() {
 		log.Err(err).Msg(loadConfigurationMessage)
 		os.Exit(ExitStatusConfiguration)
 	}
-
+	log.Error().Msg("TEST ERROR FOR NOTIFICATION SERVICE")
```
Comment (epapbak) [inline cmd/ccx-notification-service/main.go:87]: remove, there will be errors already, so it's not needed

Comment (Jakub007d) [reply cmd/ccx-notification-service/main.go]: Ok removed

File: differ/differ.go (hunk 1 above)
Comment (Jakub007d) [reply differ/differ.go]: Ok removed

**Review by epapbak — APPROVED** [review]

**Review by matysek — COMMENTED** [review]
/lgtm

**Pull request merged**